test/cli/suggest-kernel/test.out

```diff
@@ -38,18 +38,6 @@ test/cli/suggest-kernel/suggest-kernel.rb:5: Method `Integer` does not exist on
     test/cli/suggest-kernel/suggest-kernel.rb:5: Insert `.new`
      .. |    Integer(0)
                    ^
-  Note:
-    Ruby uses `.new` to invoke a class's constructor
-  Autocorrect: Use `-a` to autocorrect
-    test/cli/suggest-kernel/suggest-kernel.rb:5: Insert `.new`
-     .. |    Integer(0)
-                   ^
-  Note:
-    Ruby uses `.new` to invoke a class's constructor
-  Autocorrect: Use `-a` to autocorrect
-    test/cli/suggest-kernel/suggest-kernel.rb:5: Insert `.new`
-     .. |    Integer(0)
-                   ^
```

Comment:
Why do we lose the error note in these errors?

Reply:
We lose repetions of it, as we used to append that note for each constant suggestion that came out of `findMemberFuzzyMatch`. Now we only see the single suggestion, which is right above the negative part of the diff.

Reply:
oh I see, thanks